README.md

```diff
@@ -24,7 +24,119 @@ $ helm install nfs-subdir-external-provisioner nfs-subdir-external-provisioner/n
     --set nfs.path=/exported/path
 ```
 
-### Without Helm
+### With Kustomize
+
+**Step 1: Get connection information for your NFS server**
+
+Make sure your NFS server is accessible from your Kubernetes cluster and get the information you need to connect to it. At a minimum you will need its hostname.   
```

Comment:
please [remove trailing spaces](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md009---trailing-spaces).

nit: hostname is not enough, the exported directory required as well :)

Reply:
Will fix the trailing space. I copied the first step from the next section since they have those in common, but I will add to it.